model_cards/bert-base-cased-README.md

```diff
@@ -1,10 +1,230 @@
 ---
+language: english
 tags:
 - exbert
-
 license: apache-2.0
+datasets:
+- bookcorpus
+- wikipedia
```

Comment:
In cases where the exact pretraining code was not open-sourced (as is the case for BERT, if I'm not mistaken), I'm assuming it is still "scientifically correct" to link to the `huggingface/nlp` versions of the datasets.

Right @thomwolf?

Reply:
Yes, I confirm. We are now hosting a version of the `bookcorpus` (thanks to @mariamabarham + community) which is awesome!